opt_adapt/opt.py

```diff
@@ -161,11 +152,11 @@ def line_search(forward_run, m, u, P, J, dJ, params):
         return params.lr
 
     # Perform line search
-    if disp > 0:
+    if disp > 1:
```

Comment:
Oh okay so you are hiding the line search progress by default. This is probably a good idea, thanks.